contracts/contracts/GrantRound.sol

```diff
@@ -50,16 +56,17 @@ contract GrantRound {
    * @param _minContribution Miniumum donation amount that can be made using the given donation token
    */
   constructor(
-    address _owner,
+    address _payoutAdmin,
     GrantRegistry _registry,
     IERC20 _donationToken,
     uint256 _startTime,
     uint256 _endTime,
     string memory _metaPtr,
     uint256 _minContribution
   ) {
+    owner = msg.sender;
```

Comment:
PR looks good! Only comment is that `owner` should also be a constructor arg. The `GrantRound` contracts will be deployed from `GrantRoundFactory` which means the owner would be the factory contract